.eslintrc.js

```diff
@@ -185,31 +185,42 @@ module.exports = {
             zones: [
               {
                 target: [
-                  'src/legacy/**/*',
-                  'x-pack/**/*',
-                  '!x-pack/**/*.test.*',
-                  '!x-pack/test/**/*',
+                  '(src|x-pack)/legacy/**/*',
                   '(src|x-pack)/plugins/**/(public|server)/**/*',
-                  'src/core/(public|server)/**/*',
                   'examples/**/*',
                 ],
                 from: [
                   'src/core/public/**/*',
-                  '!src/core/public/index.ts',
-                  '!src/core/public/mocks.ts',
-                  '!src/core/public/*.test.mocks.ts',
+                  '!src/core/public/index.ts', // relative import
+                  '!src/core/public/mocks',
+                  '!src/core/public/mocks.ts', // relative import
+                  '!src/core/public/*.test.mocks',
```

Comment:
do we not need both `!src/core/public/*.test.mocks` and `!src/core/public/*.test.mocks.ts`?

Reply:
It's not being used, but added.